data-serving/data-service/src/common/db-client.ts

```diff
@@ -0,0 +1,20 @@
+import { MongoClient } from 'mongodb';
```

Comment:
This is the right place to start. We should probably take a look at [mongoose](https://www.npmjs.com/package/mongoose), which is a super popular object-management library built on top of (replacing) mongodb. I haven't looked at the trade-offs, now that native MongoDB validators are a thing, but I'd be interested in your thoughts.

Reply:
yeah I started looking into it and found opinions on both sides so I decided to defer on that decision, but definitely agree that we need to look into it

(here's one piece I started reading earlier: https://medium.com/better-programming/mongoose-vs-the-mongodb-node-driver-7dc59f00a5dd)